libs/build_api_v2.py

```diff
@@ -96,7 +135,7 @@ def build_region_summary(
         riskLevels=risk_levels,
         lastUpdatedDate=datetime.utcnow(),
         locationId=region.location_id,
-        url=latest_values[CommonFields.CAN_LOCATION_PAGE_URL],
+        url=latest_values.get(CommonFields.CAN_LOCATION_PAGE_URL),
```

Comment:
basically I wanted to write testing functions on a subset of the data - and of the direct access prevents from writing targeted testing... I've probably advocated against this in the past, but since we have the test on real data i feel somewhat comfortable?